build.gradle

```diff
@@ -8,13 +8,13 @@ buildscript {
 
         gradle_plugin_version = '4.2.1'
 
-        minSdkVersion = 16
-        versionName = "3.1.0"
+        minSdkVersion = 21
```

Comment:
I am fine with upping the minSdk, but this has to become a major release bump then. just make 4.0.0 right away.